src/index.html

```diff
@@ -1,13 +1,301 @@
+                  </p>
+                </div>
+              </article>
+
+              <article class="category">
+                <div class="category__photos">
+                  <a href="#" class="category__link category__link--square">
+                    <img
+                      src="./images/Hydraulic disc brakes & Lightweight-1.png"
+                      alt="Hydraulic disc brakes & Lightweight"
+                      class="category__photo photo"
+                    >
+                  </a>
+
+                  <a href="#" class="category__link category__link--wide">
+                    <img
+                      src="./images/Hydraulic disc brakes & Lightweight-2.png"
+                      alt="Hydraulic disc brakes & Lightweight"
+                      class="category__photo photo"
+                    >
+                  </a>
+                </div>
+
+                <h3 class="category__title">Hydraulic disc brakes & Lightweight</h3>
+                <div class="grid__wrapper">
+                  <p class="category__description">
+                    Brakes with total stopping power the second you make contact.
+                    The removable battery has up to 70km battery autonomy and weighs only 2.4 kg.
+                    Lights integrated into the frame give you always-on visibility day and night.
+                  </p>
+                </div>
+              </article>
+
+              <div class="grid__wrapper">
+                <a href="#" class="categories__button">Explore</a>
+              </div>
+            </div>
+          </section>
+
+          <section class="contacts" id="contacts">
+
+            <h2 class="section-title">Contact us</h2>
+
+            <div class="contacts__wrapper">
+
+              <form class="contacts__inputs" onsubmit="this.reset(); return false;">
+                <input
+                    type="text"
+                    name="name"
+                    placeholder="Name"
+                    class="contacts__input"
+                    required
+                >
+
+                <input
+                    type="email"
+                    name="email"
+                    placeholder="Email"
+                    class="contacts__input"
+                    required
+                >
+
+                <textarea
+                  name="message"
+                  type="text"
+                  placeholder="Message"
+                  cols="30"
+                  rows="10"
+                  class="contacts__input contacts__message"
+                  required
+                ></textarea>
+
+                <button type="submit" class="contacts__button">Send</button>
+              </form>
+
+
+              <article class="contacts__information">
+                <div class="contacts__info">
+                  <p class="contacts__head">Phone</p>
+                  <a href="tel: +1 234 5555-55-55" class="contacts__text">+1 234 5555-55-55</a>
```

Comment:
Bad value tel: +1 234 5555-55-55 for attribute href on element [a](https://html.spec.whatwg.org/multipage/#the-a-element): Illegal character in scheme data: space is not allowed.